cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/Measure.java

```diff
@@ -522,6 +600,17 @@ public SelectedContained observationHasExtensionUrl() {
             return this;
         }
 
+        /**
+         * only applicable to individual reports
+         * @return
+         */
+        public SelectedContained observationHasSDECoding() {
+            var obs = (Observation) value();
```

Comment:
I'd add an assertion that value() is an Observation first.  If for some reason it's not the same instance, the failure will be more obvious.